backend/de.metas.contracts/src/main/sql/postgresql/system/50-de.metas.contracts/5699910_sys_interim_contract_default_transition.sql

```diff
@@ -0,0 +1,23 @@
+-- 2023-08-23T17:14:34.865196500Z
+INSERT INTO C_Flatrate_Transition (AD_Client_ID, AD_Org_ID, C_Calendar_Contract_ID, C_Flatrate_Transition_ID, Created, CreatedBy, DocAction, DocStatus, EndsWithCalendarYear, IsActive, IsAutoCompleteNewTerm, IsNotifyUserInCharge, Name, Processed, Processing, TermDuration, TermDurationUnit, TermOfNotice, TermOfNoticeUnit, Updated, UpdatedBy)
+VALUES (1000000, 1000000, 1000000, 540039, TO_TIMESTAMP('2023-08-23 19:14:34.858', 'YYYY-MM-DD HH24:MI:SS.US'), 100, 'CO', 'DR', 'N', 'Y', 'N', 'N', 'Default value for interim contract', 'N', 'N', 0, 'day', 0, 'day', TO_TIMESTAMP('2023-08-23 19:14:34.858', 'YYYY-MM-DD HH24:MI:SS.US'), 100)
```

Comment:
is it ok to have C_Calendar_Contract_ID=1000000 in migration scripts?

Reply:
It's our existing accounting calendar